docs/update-privacy-policy.md

```diff
@@ -0,0 +1,47 @@
+
+# Updating the Privacy Policy
+
+The [Thunderbird Privacy Policy](https://www.thunderbird.net/en-US/privacy/) is automatically generated and should not be edited by hand.
+
+To update it, you will:
+1. Create a new branch.
+2. Run the build tool with the `--downloadlegal` option.
+3. Commit the changes.
+3. Create a Pull Request.
```

Comment:
```suggestion
4. Create a Pull Request.
```

Reply:
I also feel like this point-form list is a little unnecessary.

Reply:
I think you're right - since there are so few steps, an overview isn't necessary. I'll nix it